.github/RELEASE_CHECKLIST.md

```diff
@@ -12,16 +12,17 @@
 6. Make a signed tag `vX.Y.Z` with the relevant section of the changelog using `--cleanup=verbatim`.
 7. Push it!
 8. Make [release](https://github.com/FerretDB/FerretDB/releases).
-9. Refresh
-   * `env GOPROXY=https://proxy.golang.org go install -v github.com/FerretDB/FerretDB/cmd/ferretdb@<tag>`
-   * `env GOPROXY=https://proxy.golang.org go install -v github.com/FerretDB/FerretDB/ferretdb@<tag>`
-     (expected error message is `package github.com/FerretDB/FerretDB/ferretdb is not a main package`)
-   * <https://pkg.go.dev/github.com/FerretDB/FerretDB>
-10. `task docker-local`
-11. `task docker-push` with four tags (`X.Y.Z` without leading `v` and `latest` for both ghcr.io and Docker Hub):
+9. Upload .deb and .rpm packages from the CI build for the tag.
```

Comment:
> 9. Upload .deb and .rpm packages from the CI build for the tag. 

Maybe, download it? WDYM? If upload then probably a link to upload page is needed.

Reply:
Upload it to the newly created release from the previous point

Reply:
> Upload it to the newly created release from the previous point

What is the previous point?  Should I press the button `Draft a new release`? It's not clear what to do here.

Reply:
"Draft a new release" is the point number 3. This is point number 9. The previous point is the one above, with number 8

Reply:
> "Draft a new release" is the point number 3. This is point number 9. The previous point is the one above, with number 8

Point number 8 is not clear also.
Sorry, that's not clear: where should I take the `.deb` and `.rpm`packages for uploading?

Reply:
Tag, pushed on step 7, triggers a CI build. Please see the rest of this PR.

Reply:
> Tag, pushed on step 7, triggers a CI build. Please see the rest of this PR.

* I'd better describe in point 7,  that  `.deb` and `.rpm` packages will be there as a CI result 
* so are we have to copy packages from CI to the release page? If so can we write down that on the doc? Otherwise, there are too many assumptions that are clear to you who did it a lot of times.